Log format of failed authentications

### Log format of failed authentications
The log format of failed log ins differ from succesful logins, and more importantly, they differ from what is documented in the wiki (https://github.com/dani-garcia/bitwarden_rs/wiki/Fail2Ban-Setup#pre-requisite).

Current output:
```
$ docker-compose logs bitwarden | grep User
bitwarden    | [2020-11-20 10:54:52.147][error][ERROR] Username or password is incorrect. Try again. IP: XX.XX.XX.XX. Username: example@email.com.
bitwarden    | [2020-11-20 10:55:15.178][bitwarden_rs::api::identity][INFO] User example@email.com logged in successfully. IP: XX.XX.XX.XX
```
Expected output:
```
$ docker-compose logs bitwarden | grep User
bitwarden    | [2020-11-20 10:54:52.147][bitwarden_rs::api::identity][ERROR] Username or password is incorrect. Try again. IP: XX.XX.XX.XX. Username: example@email.com.
bitwarden    | [2020-11-20 10:55:15.178][bitwarden_rs::api::identity][INFO] User example@email.com logged in successfully. IP: XX.XX.XX.XX
```

### My environment
* Bitwarden_rs version: 1.17.0
* Install method: docker

--- a/src/error.rs
+++ b/src/error.rs
@@ -191,6 +191,7 @@ impl<'r> Responder<'r> for Error {
     fn respond_to(self, _: &Request) -> response::Result<'r> {
         match self.error {
             ErrorKind::EmptyError(_) => {} // Don't print the error in this situation
+            ErrorKind::SimpleError(_) => {} // Don't print the error in this situation
             _ => error!(target: "error", "{:#?}", self),
         };
 
@@ -210,9 +211,11 @@ impl<'r> Responder<'r> for Error {
 #[macro_export]
 macro_rules! err {
     ($msg:expr) => {{
+        error!("{}", $msg);
         return Err(crate::error::Error::new($msg, $msg));
     }};
     ($usr_msg:expr, $log_value:expr) => {{
+        error!("{}. {}", $usr_msg, $log_value);
         return Err(crate::error::Error::new($usr_msg, $log_value));
     }};
 }

